doc/sphinxext/ipython_sphinxext/ipython_directive.py

```diff
@@ -127,7 +127,11 @@
 from sphinx.util.compat import Directive
 
 # Our own
-from IPython import Config, InteractiveShell
+try:
+    from traitlets.config import Config
+except ImportError:
```

Comment:
the fact that you have to change this is very odd. is this in their release notes? (or are we just using reallly old syntax)